lib/oca-epak/epak/client.rb

```diff
@@ -169,7 +174,20 @@ def taxation_centers_with_services_by_zipcode(opts = {})
         method = :get_centros_imposicion_con_servicios_by_cp
         response = client.call(method, message: message)
         parse_result(response, method)
-      end      
+      end
+
+      private
+      def normalize_cuit(cuit)
+        pattern = /[0-9]{2}-[0-9]{8}-[0-9]/
+
+        cuit_number = if (cuit =~ pattern).nil?
+          "%s-%s-%s" %[cuit[0..1], cuit[2..9], cuit[10]]
+        else
+          cuit
+        end
+
+        return cuit_number
```

Comment:
@cleicar No need to include the `return` here. It could just end like this:

```
  if (cuit =~ pattern).nil?
    "%s-%s-%s" %[cuit[0..1], cuit[2..9], cuit[10]]
  else
    cuit
  end
end
```